person.rb

```diff
@@ -1,17 +1,20 @@
 require_relative 'nameable'
 require_relative 'capitalize_decorator'
 require_relative 'trimmer_decorator'
+require_relative 'rental'
+require_relative 'book'
 
 class Person < Nameable
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
- [ ] The `rentals` variable should have `attr_accessor`, not `attr_reader`. Kindly do that.